### ISSUE TITLE
Set Mobile Breakpoints - More work section

### ISSUE THREAD

**Issue by jonpekk** [codeforboston/boston-liquor-license-tracker#222]
**Requirements not yet completed**:
- card should be full width and scale as screen gets smaller (right now its fixed and overflows)
- add black stroke around all text for readability 
- ensure padding, margin, etc. matches the designs

<img width="847" height="873" alt="Image" src="https://github.com/user-attachments/assets/1f0c900e-2913-42f4-a9e0-7f906759eedc" />
